new/LIT6280PrayerGeorge.xml

```diff
@@ -74,7 +74,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             ኀያል፡ መስተጋድል፡ ኀያል፡ ኮከበ፡ ክብር፡ ኀያል፡ ገባሬ፡ ተአምር ፡ ጊዮርጊስ፡ ቅዱ<choice>
                                 <sic>ሰ</sic>
                                 <corr resp="SH">ሱ</corr>
-                            </choice>፡ ለእግዚአብሔር፡ በእንተ፡ ጊዮርጊስ ፡  ሰማዕትከ ፡  ምህላነ ፡ ስማዕ፡ በዕዝንከ፡ ፫ ጊዜ፡ በል፡
+                            </choice>፡ ለእግዚአብሔር፡ በእንተ፡ ጊዮርጊስ፡ ሰማዕትከ፡ ምህላነ ፡ ስማዕ፡ በዕዝንከ፡ ፫ ጊዜ፡ በል፡
```

Comment:
The last space before ፡ in ምህላነ-፡
I'm sorry to be sooo much pedantic :(

Reply:
Massimo, may I ask you to delete or add "space" directly here in GitHub next time, since I am obviously blind to it. We once agreed that such very minor and obvious (but impoortant!) corrections could be done directly on GitHUb.

Reply:
I'm sorry for my requests, I'm a bit unfamiliar with direct corrections on Github on other files. I'll try to do it next time! :)